static/js/markdown.js

```diff
@@ -303,12 +303,12 @@ function handleTimestamp(time) {
     const istimevalid = !(timeobject === null || !timeobject.isValid());
 
     // Generate HTML
-    let timestring = '<span class="timestamp"';
-    if (istimevalid) {
-        timestring += ' data-timestamp="' + timeobject.unix() + '"';
-    }
-    timestring += '>' + _.escape(time) + '</span>';
-    return timestring;
+
+    // Use html5 <time> tag for valid timestamps.
+    const tag = istimevalid ? 'time' : 'span';
+    // If we have a valid time, render it without milliseconds.
+    const timestring = istimevalid ? ` timestamp=${timeobject.toISOString().split('.')[0] + "Z"}` : '';
```

Comment:
We could rewrite this block as:

```js
let tag = 'span';
let timestring = '';
if (istimevalid) {
    tag = 'time';
    timestring = timeobject.toISOString().split('.')[0] + "Z";
}
```

I think both are equally readable for someone who is looking to make any changes here, so I went with the ternary version.

Reply:
We generally prefer to avoid ternary operations.

Reply:
Pushed without ternary.